internal/common/common.go

```diff
@@ -248,3 +251,50 @@ func GetHostNTPSources(db *gorm.DB, host *models.Host) (string, error) {
 	}
 	return infraEnv.AdditionalNtpSources, nil
 }
+
+// Returns a list of certificate objects extracted from a byte array
+// The certificates within the byte array must be PEM encoded
+func ParsePemCerts(pemCerts []byte) ([]x509.Certificate, bool) {
+	var certs []x509.Certificate
+	for len(pemCerts) > 0 {
+		var block *pem.Block
+		block, pemCerts = pem.Decode(pemCerts)
+		if block == nil {
+			return certs, true
+		}
+		if block.Type == "CERTIFICATE" && len(block.Headers) == 0 {
+			cert, err := x509.ParseCertificate(block.Bytes)
+			if err != nil {
+				return nil, false
+			}
+			certs = append(certs, *cert)
+		}
+	}
+	return certs, true
+}
+
+// Verifies the full CA Chain contained in a byte array.
```

Comment:
The doc text of a function always needs to start with the function name.
 // VerifyCaBundle verifies the full CA Chain contained in a byte array.
....